src/aegis_ai/toolsets/tools/kernel_cves/__init__.py

```
@@ -257,7 +257,7 @@ async def kernel_cve_lookup(cve_id: CVEID) -> LINUXCVEToolResponse:
     and parsing the relevant files for context.
     """
     try:
-        subprocess.run(["git", "--version"], check=True, capture_output=True)
+        subprocess.run(["git", "--version"], check=True, capture_output=True)  # noqa: ASYNC221
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)